nurseconnect/models.py

```diff
@@ -0,0 +1,102 @@
+from django.contrib.auth.hashers import make_password, check_password
+from django.contrib.auth.models import User
+from django.db import models
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from nurseconnect.constants import GENDERS
+from molo.commenting.models import MoloComment
+from wagtail.contrib.settings.models import BaseSetting
+from wagtail.contrib.settings.registry import register_setting
+from wagtail.wagtailadmin.edit_handlers import FieldPanel
+
+
+class NurseConnectUserProfile(models.Model):
+    user = models.OneToOneField(
+        User,
+        related_name="nurse_connect_profile",
+        primary_key=True
+    )
+    gender = models.CharField(
+        max_length=1,
+        choices=GENDERS,
+        blank=True,
+        null=True
+    )
+    mobile_number = models.CharField(
+        max_length=15
+    )
+    staff_number = models.CharField(
+        max_length=15
+    )
+    facility_code = models.CharField(
+        max_length=15
+    )
+    security_question_1_answer = models.CharField(
+        max_length=128,
+        null=True
+    )
+    security_question_2_answer = models.CharField(
+        max_length=128,
+        null=True
+    )
+
+    # based on django.contrib.auth.models.AbstractBaseUser set_password &
+    # check_password functions (takes plain text, generates hash for database)
+    def set_security_question_1_answer(self, raw_answer):
```

Comment:
The setter is passed in and will be called when the match was successful and it'll then generate a new hash for the security question.